C++/Maths/Josephus.cpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <list>
+using namespace std;
+
+// Josephus Problem Algorithm:
+// 1. Create a circular list to represent the circle of people.
+// 2. Initialize the list with people numbered from 1 to n.
+// 3. Start with an iterator pointing to the first person.
+// 4. Repeat the elimination process until only one person remains:
+//     a. Move the iterator k-1 steps in a circular manner.
+//     b. Erase the person at the current position.
+//     c. Update the iterator to the next position after the elimination, wrapping around if necessary.
+// 5. Return the position of the last remaining person.
+
+int josephus(int n, int k) {
+    std::list<int> circle;

```

Comment:
Why did you use ```std``` when you have already written ```using namespace std;``` at line 3

Reply:
For list function. Sir did not know about the list function that belongs to the namespace so I mentioned